docker/nginx/config/nginx.conf.erb

```diff
@@ -0,0 +1,152 @@
+<%# params: -%>
+<%# - fake_hosts            string[] - when set only fake vhosts will be created - used for certbot initial run-%>
+<%# - server_name           string   - main domain/server_name for the wewbie -%>
+<%# - redirect_server_names string[] - all domains that will redirect to server_name and server_name (to redirect http->https) -%>
+<%# - https_cert_name       string   - used in /etc/letsencrypt/NAME, if null - only http will be enabled -%>
+<%
+  fake_hosts             = (fake_hosts            or ENV['FAKE_HOSTS']);
```

Comment:
You might want to write this as `fake_hosts ||= ENV['FAKE_HOSTS'], same as the rest of the variables below